docs/extensions.md

```diff
@@ -35,6 +35,33 @@ checking to ensure data is consistent, ie. usage data with `n` incremented in
 5 necessarily means `n` parent, if existing, should be specified with a minimum
 of 5, not counting any other potential children.
 
+### list_app_keys (integer)
+
+This extension just outputs an XML `app_keys` section in authorization responses
+so that application keys registered to it they will be listed there with the
+following format:
+
+> \<app_keys app="app_id" svc="service_id"\>
+>   \<key id="app_key1"/\>
```

Comment:
Why the `id` field ?

Reply:
My question was more about needing the attribute. I guess you did it like this because it allows to add more attributes in the future, although I don't think that's going to happen.
It's fine :+1:

Reply:
Well, not exactly - the attributes indeed help extensibility. But I used it to be able to have a self-closing tag rather than:

> \<key\>some_app_key\</key\>

This usually helps parsing a bit and is less verbose. In this case attributes can still be added later, so that's not a factor.

Reply:
ok :+1: